lib/util/print_separator.h

```diff
@@ -8,6 +8,6 @@
  * Print ${separator} to ${stream} if ${print_nulls} is zero; otherwise,
  * print ${num_nulls} '\0'.
  */
-int print_separator(FILE *, char *, int, int);
+int print_separator(FILE *, const char *, int, int);
```

Comment:
My bad.  In #643, I was so fixated on whether we'd allow `#define print_sep` that I missed that the API should take `const`.

Reply:
Can you put split this into a separate commit?